agent/src/index.ts

```diff
@@ -318,42 +318,53 @@ function initializeDatabase(dataDir: string) {
     }
 }
 
+// also adds plugins from character file into the runtime
 export async function initializeClients(
     character: Character,
     runtime: IAgentRuntime
 ) {
-    const clients = [];
-    const clientTypes =
+    // each client can only register once
+    // and if we want two we can explicitly support it
+    const clients:Record<string, any> = {};
```

Comment:
Thanks so much for doing this! Appreciate how hard you are grinding! Looks like a space is missing after the colon for all the type declarations here